src/test/java/Pages/BasePage.java

```diff
@@ -0,0 +1,26 @@
+package Pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.interactions.Actions;
+import javax.swing.*;
+import java.time.Duration;
+
+public class BasePage {
+    public WebDriver driver;
+    public WebDriverWait wait;
+    Actions actions;
+    String playlistName = "LinaPelo2323";
+
+    public BasePage(WebDriver givenDriver) {
+        driver = givenDriver;
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        actions = new Actions(driver);
+    }
+    public void doubleClick(WebElement locator){
+ wait.until(ExpectedConditions.elementToBeClickable(locator));
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)